ror/spec/requests/tags_request_spec.rb

```diff
@@ -0,0 +1,205 @@
+require 'rails_helper'
+
+RSpec.describe "Tag requests", type: :request do
+    let(:existed_tag_name) { "existedTagName" }
+    let(:invalid_tag_id) { "0" }
+    let(:invalid_article_id) { "0" }
+    @existed_tags_length = nil
+    @existed_tag_object = nil
+    @existed_tag_id = nil
+    before do
+        post "/tags", params: { :name => existed_tag_name }
+        @existed_tag_object = JSON.parse(response.body)["message"]
```

Comment:
I got an error saying that
```
`let` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
```
I guess what I want to achieve here is to assign the tag object returned from `post '/tags` endpoint to the variable `@existing_tag_object` so that in all subsequent tests, I can use it to do the assertion if needed.